filament/backend/src/vulkan/VulkanContext.cpp

```diff
@@ -189,6 +189,7 @@ void createLogicalDevice(VulkanContext& context) {
     VkPhysicalDeviceFeatures enabledFeatures {
         .textureCompressionETC2 = supportedFeatures.textureCompressionETC2,
         .textureCompressionBC = supportedFeatures.textureCompressionBC,
+        .samplerAnisotropy = supportedFeatures.samplerAnisotropy,
```

Comment:
apparently the failure on window is because the designated initializers must be in order